examples/Remarketing/UploadOfflineConversions.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Remarketing;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\V2\GoogleAdsClient;
+use Google\Ads\GoogleAds\Lib\V2\GoogleAdsClientBuilder;
+use Google\Ads\GoogleAds\Lib\V2\GoogleAdsException;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
+use Google\Ads\GoogleAds\Util\V1\ResourceNames;
+use Google\Ads\GoogleAds\V2\Services\ClickConversion;
+use Google\Ads\GoogleAds\V2\Errors\GoogleAdsError;
+use Google\Ads\GoogleAds\V2\Services\ClickConversionResult;
+use Google\Ads\GoogleAds\V2\Services\UploadClickConversionsResponse;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\DoubleValue;
+use Google\Protobuf\StringValue;
+
+/**
+ * This code example imports offline conversion values for specific clicks to
+ * your account. To get Google Click ID for a click, use the "click_view"
+ * resource of report. To set up a conversion action, run the
+ * AddConversionAction.php example.
+ */
+class UploadOfflineConversions
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    const CONVERSION_ACTION_ID = 'INSERT_CONVERSION_ACTION_ID_HERE';
+    const GCL_ID = 'INSERT_GCL_ID_HERE';
```

Comment:
Based on several sources, it seems the most frequently used abbreviation is just "gclid", not "gclId".
I know we were not consistent in the past, but let's fix this here by using "gclid".
So, this const should be named GCLID, since gcl and id are in the same word.